src/code/PreRender.c

```diff
@@ -91,158 +101,244 @@ void func_8016FF90(PreRender* this, Gfx** gfxp, void* buf, void* bufSave, s32 en
     gDPSetScissor(gfx++, G_SC_NON_INTERLACE, 0, 0, this->width, this->height);
 
     Prerender_DrawBackground2D(&gfx, buf, 0, this->width, this->height, G_IM_FMT_RGBA, G_IM_SIZ_16b, G_TT_NONE, 0, 0.0f,
-                               0.0f, 1.0f, 1.0f, 0xB);
+                               0.0f, 1.0f, 1.0f, BG2D_FLAGS_1 | BG2D_FLAGS_2 | BG2D_FLAGS_LOAD_S2DEX2);
     gDPPipeSync(gfx++);
     gDPSetColorImage(gfx++, G_IM_FMT_RGBA, G_IM_SIZ_16b, this->width, this->fbuf);
 
     *gfxp = gfx;
 }
 
-void func_80170200(PreRender* this, Gfx** gfxp, void* buf, void* bufSave) {
-    func_8016FF90(this, gfxp, buf, bufSave, 255, 255, 255, 255);
+/**
+ * Copies `fbuf` to `fbufSave`, discarding the alpha channel and leaving the rgb channel unchanged
+ */
+void func_80170200(PreRender* this, Gfx** gfxp, void* fbuf, void* fbufSave) {
+    func_8016FF90(this, gfxp, fbuf, fbufSave, 255, 255, 255, 255);
```

Comment:
Perhaps it is fine, but looks like in MM, this doesn't always use fbuf and fbufSave.
```c
func_80170200(&play->pauseBgPreRender, &gfxTemp, zbuffer, spA4);
```
in `Actor_DrawLensActors`

Reply:
That's true, I don't feel confident about the comment so I just removed it for now